src/util/system.h

```diff
@@ -129,12 +129,23 @@ class ArgsManager
 {
 public:
     enum Flags {
```

Comment:
Could the underlying type be specified?
```suggestion
     enum Flags : unsigned int {
```
Ref: https://github.com/bitcoin/bitcoin/pull/16097#discussion_r310193132